### PR TITLE
Setup actions workflow for testing and building artifacts

### PR DESCRIPTION
**Pull request by chloe-zh**
*Issue #, if available:*

*Description of changes:*

build example: https://github.com/chloe-zh/notifications/actions/runs/324646312

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/test-and-build-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Notifications Test and Build
```
Comment (akbhatta) [inline .github/workflows/test-and-build-workflow.yml:18]: license header?
Also "---" at the beginning of file

File: .github/workflows/test-and-build-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Create Artifact Path
+        run: |
+          mkdir -p notifications-build
+          cp -r ./build/distributions/*.zip notifications-build/
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: notifications
+          path: notifications-build
```
Comment (akbhatta) [inline .github/workflows/test-and-build-workflow.yml]: nit: new line at the end of file

**Review by akbhatta — APPROVED** [review]

File: .github/workflows/test-and-build-workflow.yml (hunk 2 above)
Comment (chloe-zh) [reply .github/workflows/test-and-build-workflow.yml]: done

File: .github/workflows/test-and-build-workflow.yml (hunk 1 above)
Comment (chloe-zh) [reply .github/workflows/test-and-build-workflow.yml:18]: done

**Pull request merged**